src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -822,12 +563,12 @@ def _runtool_hermetic(self, main, tool_name, args, distribution, tgt=None, input
   # execution requests.
   def _runtool_nonhermetic(self, parent_workunit, classpath, main, tool_name, args, distribution):
     result = self.runjava(classpath=classpath,
```

Comment:
My editor appears to format this differently than yours, and usually I would expect that if the first argument is indented, the others are indented at the same level. This isn't always followed because some editors require you to manually tab, but if we're trying to reduce whitespace noise in future diffs, I would probably also put the `classpath=` argument on the following line.

Reply:
Yeah, I'd be up for bumping the first arg down to the next line :)